src/Sylius/Bundle/ApiBundle/Resources/config/serialization/ChannelPricing.xml

```diff
@@ -16,10 +16,6 @@
             xsi:schemaLocation="http://symfony.com/schema/dic/serializer-mapping https://symfony.com/schema/dic/serializer-mapping/serializer-mapping-1.0.xsd"
 >
     <class name="Sylius\Component\Core\Model\ChannelPricing">
-        <attribute name="id">
-            <group>admin:product_variant:read</group>
```

Comment:
Why are you removing an id?

Reply:
This field is non-operational. During PUTs you can set basically anything there, API Platform will use @id anyway. Maybe leaving this field would make sense if we supported Content-Type: application/json, but currently, for a reason unknown to me, it's bugged and trying to update resources with plain JSON (application/json header) is limited (and I wonder if it's needed). 
I've removed this field to discuss whether we shouldn't get rid of this field in other places as well (and if not why not)